### ISSUE TITLE
Samurai Simple Job Gauge - Kenki bar scales from 5 to 100

### ISSUE THREAD

**Issue by Qunka724** [skotlex/ffxiv-material-ui#169]
Hi,

This is regarding option 04_gaugebar-frame-silver. For Samurai, the Kenki bar doesn't fill properly from 0. When you gain 5 kenki, it stays at the bottom. Then, it scales from 5 to 100%.

[Here's a picture](https://i.imgur.com/hWqGUKU.png)

**Comment by skotlex** [comment]
I already knew that, but I couldn't help it. It's because of the decorations on the left and right sides, and there are structural limitations, so I had to give up the gauge 5 mark on the left or right. And as a result of the test, I eventually gave up the left because the awkwardness is strong if I give up the gauge 5 on the right.

In order to solve this, I have to get rid of the decorations on the left and right sides, and I don't want it. I hope you understand.
